cypress/support/pages/contactForm.pageObject.js

```diff
@@ -19,11 +19,11 @@ class ContactFormPageObject extends PageObject {
     return cy.contains('.btn', 'Send message');
   }
 
-  typeEmail(email) {
+  typeEmail() {
     this.emailField.type(email, { force: true });
```

Comment:
The 'typeEmail' method is missing the 'email' parameter in its definition. You should pass the 'email' parameter to this method to ensure it functions correctly.